coderedcms/templates/coderedcms/blocks/image_block.html

```diff
@@ -1,6 +1,6 @@
 {% load wagtailimages_tags %}
 
-{% image self.image max-1000x1000 as self_image %}
+{% image self.image original as self_image %}
```

Comment:
Ditto on the max parameter. I think the problem that was causing you trouble before was the `w-100` class below.

Maybe we should use `max-2000x2000` as a safeguard, but replace `w-100` with `img-fluid` to ensure the image will scale, but not be forced to 100% width if it is smaller?